cpp/daal/BUILD

```diff
@@ -65,8 +65,11 @@ daal_module(
 
 daal_module(
     name = "services",
-    srcs = glob(
-        ["src/services/**/*.cpp", "src/externals/**/*.cpp"],
+    srcs = glob([
+            "src/services/**/*.cpp",
+            "src/externals/**/*.cpp",
+            "src/algorithms/*.cpp",
```

Comment:
Not all algorithms, but only service functionality in algorithms. The problem is `src/algorithms/(*.h|*.cpp)` defines such classes as `Argument`, `HashTable`, etc., which are mostly services rather than algorithms (don't ask my why they are in algorithms). In fact functionality in `src/services` depends on it.

Reply:
Ok, it's fine